controllers/api.js

```diff
@@ -312,7 +318,7 @@ router.post('/payinvoice', async function (req, res) {
       let inv = {
         payment_request: req.body.invoice,
         amt: info.num_satoshis, // amt is used only for 'tip' invoices
-        fee_limit: { fixed: Math.floor(info.num_satoshis * 0.005) + 1 },
```

Comment:
feels like `0.005`  is a bug and actually `0.01` was intended here. thoughts?

Reply:
its seldom hit ? (a amt less invoice ?) anyway forwardFee in its place works for me altho now is set in this PR as internalFee